test/mako/config/benchmark.go

```diff
@@ -25,39 +25,44 @@ import (
 
 	"github.com/golang/protobuf/proto"
 	mpb "github.com/google/mako/spec/proto/mako_go_proto"
+	"github.com/pkg/errors"
 )
 
 const koDataPathEnvName = "KO_DATA_PATH"
+const configMako = "/etc/config-mako"
 
 // MustGetBenchmark wraps getBenchmark in log.Fatalf
-func MustGetBenchmark() (*string, *string) {
-	benchmarkKey, benchmarkName, err := getBenchmark()
+func MustGetBenchmark() *mpb.BenchmarkInfo {
+	bench, err := getBenchmark()
 	if err != nil {
-		log.Fatalf("unable to determine benchmark_key: %v", err)
+		log.Fatalf("unable to determine benchmark info: %v", err)
 	}
-	return benchmarkKey, benchmarkName
+	return bench
 }
 
 // getBenchmark fetches the appropriate benchmark_key for this configured environment.
-func getBenchmark() (*string, *string, error) {
+func getBenchmark() (*mpb.BenchmarkInfo, error) {
 	// Figure out what environment we're running in from the Mako configmap.
 	env, err := getEnvironment()
 	if err != nil {
-		return nil, nil, err
+		return nil, err
 	}
 	// Read the Mako config file for this environment.
 	data, err := readFileFromKoData(env + ".config")
 	if err != nil {
-		return nil, nil, err
+		data, err = ioutil.ReadFile(filepath.Join(configMako, env+".config"))
+		if err != nil {
+			return nil, errors.Wrap(err, "cannot load both from kodata and from config mako config map")
```

Comment:
Maybe this should return both the kodata `err` and the configmap `err`. Otherwise the message of the kodata `err` will always be hidden.